packages/lib-classifier/src/store/DataVisAnnotatingStore.js

```diff
@@ -0,0 +1,37 @@
+import { types } from 'mobx-state-tree'
```

Comment:
I think missing a few imports?
```
import { autorun } from 'mobx'
import { addDisposer, getRoot, onAction, types } from 'mobx-state-tree'
```

Reply:
Thanks. I have no idea how this got messed up in my last commit, but they had to be there at one point for this to be working when I first opened the PR. 🤷‍♀️